pages/galleri/index.tsx

```diff
@@ -0,0 +1,228 @@
+  user-select: none;
+`;
+
+const Title = styled('h1')`
+  font-weight: bold;
+  font-smoothing: antialiased;
+  font-size: 3rem;
+  margin-bottom: 1rem;
+`;
+
+export default function Test() {
+  const [slideNumber, setSlideNumber] = useState(0);
+  const [openModal, setOpenModal] = useState(false);
+
+  const handleOpenModal = (index: number) => {
+    setSlideNumber(index);
+    setOpenModal(true);
+  };
+
+  // Close Modal
+  const handleCloseModal = () => {
+    setOpenModal(false);
+  };
+
+  // Previous Image
+  const prevSlide = () => {
+    slideNumber === 0
+      ? setSlideNumber(images.length - 1)
+      : setSlideNumber(slideNumber - 1);
+  };
+
+  // Next Image
+  const nextSlide = () => {
+    slideNumber + 1 === images.length
+      ? setSlideNumber(0)
+      : setSlideNumber(slideNumber + 1);
+  };
+
+  const StyledButton = ({
+    name,
+    top = '50%',
+    right = '40px',
+    left,
+    onClick,
+  }: {
+    name: string;
+    top?: string;
+    right?: string;
+    left?: string;
+    onClick: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  }) => {
+    return (
+      <>
+        {/* eslint-disable-next-line*/}
+        {/* @ts-ignore*/}
+        <ion-icon
+          onClick={onClick}
+          style={{
+            position: 'fixed',
+            fontSize: '65px',
+            color: itdageneBlue,
+            top: top,
+            right: right,
+            left: left,
+          }}
+          name={name}
+        />
+      </>
+    );
+  };
+
+  const changeImageOnKey = useCallback(
+    (event) => {
+      if (event.key === 'Escape') {
+        handleCloseModal();
+      }
+      if (event.key === 'ArrowRight') {
+        nextSlide();
+      }
+      if (event.key === 'ArrowLeft') {
+        prevSlide();
+      }
+    },
+    [slideNumber]
+  );
+
+  useEffect(() => {
+    document.addEventListener('keydown', changeImageOnKey);
+    return function cleanup() {
+      document.removeEventListener('keydown', changeImageOnKey);
+    };
+  });
+
+  return (
+    <>
+      <Layout noLoading responsive>
+        {openModal && (
+          <SliderWrap>
+            <StyledButton
+              name="close-circl d-outline"
+              top="40px"
+              onClick={handleCloseModal}
+            />
+            <StyledButton
+              name="arrow-back-outline"
+              left="40px"
+              onClick={prevSlide}
+            />
+            <StyledButton name="arrow-forward-outline" onClick={nextSlide} />
+            <FullScreenImageDiv onClick={handleCloseModal}>
+              <FullScreenImage
+                src={`https://cdn.itdagene.no/${images[slideNumber]}`}
+                alt={images[slideNumber]}
+                width={850}
+                height={330}
+              />
+            </FullScreenImageDiv>
+          </SliderWrap>
+        )}
+
+        <Title>Galleri</Title>
+        <GalleryWrap>
+          {images.map((slide, index) => {
+            return (
```

Comment:
You can drop the return and have it be implicit here

Reply:
Yes. Slipped my mind.